infra/minervaStack.js

```diff
@@ -0,0 +1,128 @@
+const cdk = require( 'aws-cdk-lib' );
+const apiGateway = require( 'aws-cdk-lib/aws-apigateway' );
+const iam = require( 'aws-cdk-lib/aws-iam' );
+const logs = require( 'aws-cdk-lib/aws-logs' );
+const lambda = require( 'aws-cdk-lib/aws-lambda' );
+const awsEvents = require( 'aws-cdk-lib/aws-events' );
+const awsEventsTargets = require( 'aws-cdk-lib/aws-events-targets' );
+const { NodejsFunction } = require('aws-cdk-lib/aws-lambda-nodejs');
+
+const path = require('path');
+
+class MinervaStack extends cdk.Stack {
+    /**
+     *
+     * @param {Construct} scope
+     * @param {string} id
+     * @param {StackProps=} props
+     */
+    constructor ( scope, id, props ) {
+        super( scope, id, props );
+
+        const { deployEnv } = props
+
+        // Default configuration for Lambda functions
+        const lambdaFnProps = {
```

Comment:
The general configuration common to all of minerva's lambda functions. Let me know if see anything that's off.